README.md

```diff
@@ -84,27 +84,19 @@ The Contacts should be first included into a specific User Groups and then the c
 
 ### Tags Propagation Policies
 
-#### DownTree propagation policy
+When down-tree propagation policy is selected, all products inherit the Tag propagated to the parent categories.
 
-When down-tree propagation policy is selected, all products inherit the Tag assigned to the parent categories.
-In order to apply this type of propagation policy to Catalog categories and products, do the following:
+When the up-tree policy propagation is applied, the Product category will inherit the Product tag, i.e. if a tag propagated to a specific product in the Category, it will be propagated to the Product Category as well.
```

Comment:
Here "assigned" is correct:
 i.e. if a tag **assigned** to a specific product in the Category, it will be propagated to the Product Category as well.

Reply:
ок!